### DISCUSSION TITLE
New Ideas

### DISCUSSION THREAD

**Discussion by GoofyAhhScratch** [ScratchAddons/ScratchAddons#5434] [Ideas]
Maybe we can have a (invite everybody on scratch) feature on scratch it can be use for trolling and for scratchcat!
Whats your idea?

**Comment by BroJac5246** [comment]
A few issues:
 1. The user might automatically get banned for spamming.
 2. That's just rude
 3. It would take forever because there are so many Scratch users.

**Comment by GrahamSH-LLK** [comment]
Impossible, not useful, and trolling (you even say it yourself) suggestion.
Locked.
